Android/app/src/main/java/app/intra/sys/IntraVpnService.java

```diff
@@ -65,8 +66,12 @@
   private static final String WARNING_CHANNEL_ID = "warning";
   private static final String NO_PENDING_CONNECTION = "This value is not a possible URL.";
 
+  // Reference to the singleton VpnController, for convenience
+  private static final VpnController controller = VpnController.getInstance();
```

Comment:
nit: consider renaming to `vpnController` to make the annotations more explicit

Reply:
Done